README.md

```diff
@@ -1,2 +1,136 @@
-# airflow-provider-skypilot
-SkyPilot provider for Apache Airflow
+<p align="center">
+    <img alt="Airflow" src="https://cwiki.apache.org/confluence/download/attachments/145723561/airflow_transparent.png?api=v2" width="60" />
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/skypilot-org/skypilot/master/docs/source/images/skypilot-wide-dark-1k.png">
+        <img alt="SkyPilot" src="https://raw.githubusercontent.com/skypilot-org/skypilot/master/docs/source/images/skypilot-wide-light-1k.png" width=240>
+    </picture>
+</p>
+<h1 align="center">
+  Apache Airflow Provider for SkyPilot
+</h1>
+  <h3 align="center">
+  A provider you can install into your Airflow environment to run SkyPilot tasks.
+</h3>
+<br/>
+
+## Operators
+
+This provider includes the `SkyTaskOperator` which creates SkyPilot clusters, runs the task
+defined in the SkyPilot YAML, and terminates the cluster upon completion.
+It integrates with Airflow connections so that you can use your existing credentials
+with your SkyPilot tasks.
+
+More operators like `SkyJobOperator` are in the roadmap, so stay tuned for that as well.
+
+## Installation
+
+You can install this package on top of an existing Airflow deployment via `pip install airflow-provider-skypilot`. For the minimum Airflow version supported, see [Requirements](#requirements) below.
+
+You should be able to see `airflow-provider-skypilot` on the Providers page
+upon successful installation.
+
+<p align="center">
+    <img alt="Airflow variables" src="https://i.imgur.com/YFSdzKz.png" width="720">
+</p>
+
+## Configuration and Usage
+
+1. Deploy a SkyPilot [remote API Server](https://docs.skypilot.co/en/latest/reference/api-server/api-server.html#remote-api-server-multi-user-teams)
+
+2. Set the `SKYPILOT_API_SERVER_ENDPOINT` variable in Airflow to point to your remote API Server endpoint
+
+    <p align="center">
+        <img alt="Airflow variables" src="https://i.imgur.com/rr7SfFP.png" width="720">
+    </p>
+
+3. [Optional] Create connections in Airflow to store your cloud credentials. Today, we support AWS and GCP connections.
+
+    <p align="center">
+        <img alt="Airflow connections" src="https://i.imgur.com/9VbD44X.png" width="720">
+    </p>
+    <p align="center">
+        <img alt="Airflow GCP connection" src="https://i.imgur.com/meHEw8w.png" width="720">
+    </p>
+
+4. Import `SkyTaskOperator`, and use it in your Airflow DAG.
+
+    ```python
+    from airflow import DAG
+
+    from skypilot_provider.operators import SkyTaskOperator
+
+    with DAG(...) as dag:
+        ...
+        task = SkyTaskOperator(
+            task_id="data_preprocess",
+            base_path="/opt/airflow", # This can point to a git repository too
+            yaml_path="data_preprocessing.sky.yaml",
+        )
+        ...
```

Comment:
the example should have a example similar to https://docs.skypilot.co/en/latest/examples/orchestrators/airflow.html

Reply:
Updated